src/app/core/models/sparque-category-tree/sparque-category-tree.interface.ts

```diff
@@ -0,0 +1,11 @@
+import { SparqueCategory } from 'ish-core/models/sparque-category/sparque-category.interface';
+
+export interface SparqueCategoryTreeResponse {
```

Comment:
the interface should always match the file pattern name - if it is an interface (and not a model) it is clear it is a response - so the name of this interface should be SparqueCategoryTree